File: my-folder/problems/running_sum_of_1d_array/solution.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        n = len(nums)
        #sum1 = 0
        res = [nums[0]]
        for i in range(1,n):
            res.append(res[i-1]+nums[i])
        return res

File: my-folder/problems/running_sum_of_1d_array/solution.java
class Solution {
    public int[] runningSum(int[] nums) {
        int tempSum = nums[0];
        for(int i= 1; i<nums.length;i++){
            tempSum = tempSum + nums[i];
            nums[i] = tempSum;
            
            
        }
        return nums;
    }
}